homeassistant/components/homematicip_cloud/device.py

```diff
@@ -93,7 +94,8 @@ def icon(self) -> Optional[str]:
     @property
     def device_state_attributes(self):
         """Return the state attributes of the generic device."""
-        attr = {ATTR_MODEL_TYPE: self._device.modelType}
+        attr = {ATTR_MODEL_TYPE: self._device.modelType, ATTR_ID: self._device.id}
```

Comment:
If it's a real unique ID, why would unique ID include the class name ?

Reply:
@balloob "my" ID refers to a physical device with a unique sgtin/id. I think the unique id you refer to is a unique hass device type belonging to that physical device. But there can be more of those all pointing to that one physical device.

Reply:
@balloob The description in my PR should have better described that I guess